app/views/users/show.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Here is a single user</h1>
\ No newline at end of file
+<body>
+  <%= render "partials/user_container", user: @users_find %>
+        <article>
+            <h3>Bio</h3>
+            <p><%= @users_find.bio %></p>
+        </article>
+
+        <article>
+            <h3>Post Title: <%= @post.title %></h3>
+            <p><%= @post.text %></p>
+            <div class="block">
+                <p>Comments: <%= @post.comments_counter %></p>
+                <p>Likes: <%= @post.likes_counter %></p>
+            </div>
+        </article>
```

Comment:
- [ ] It would be nice if you use the `most_recent_post` method to show recent three posts here to make your project do things automatically :blush: